src/AccessibilityInsights.Extensions.GitHub/ConfigurationModelControl.xaml.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System;
+using AccessibilityInsights.Extensions.Interfaces.IssueReporting;
+
+namespace AccessibilityInsights.Extensions.GitHub
+{
+    /// <summary>
+    /// Interaction logic for ConfigurationModel.xaml
+    /// </summary>
+    public partial class ConfigurationModel : IssueConfigurationControl
+    {
+        public string Config { get; set;}
+        public ConfigurationModel()
+        {
+            InitializeComponent();
+        }
+
+        /// <summary>
+        /// Called when save button clicked.
+        /// </summary>
+        /// <returns>The extension’s new configuration, serialized</returns>
+        public override string OnSave()
+        {
+            this.Config = this.tbURL.Text;
+            return Config;
+        }
+
+        /// <summary>
+        /// Called when settings page dismissed
+        /// </summary>
+        public override void OnDismiss()
+        {
+            this.tbURL.Text = this.Config;
+        }
+
+        public void TextChange_UpdateSaveButton(object sender, EventArgs e)
+        {
+            if (this.Config!= null && !this.Config.Equals(this.tbURL.Text))
```

Comment:
We can refactor this to `canSave = this.Config!= null && !this.Config.Equals(this.tbURL.Text)`

Reply:
Why stop there? :)

canSave = !this.Config.Equals(this.tbURL.Text);

---
In reply to: [267137758](https://github.com/Microsoft/accessibility-insights-windows/pull/184#discussion_r267137758) [](ancestors = 267137758)

Reply:
If config is null, we may run into issues